### PR TITLE
Fix 403 issue corner-case

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

File: src/app/layout/components/app-layout/app-layout.component.spec.ts [hunk 1]
```diff
@@ -63,6 +63,8 @@ describe('AppLayoutComponent', () => {
 
   const profileService = {
     get: () => jest.fn(),
+    getUnapprovedUser: () => of(),
+    userNotApproved: false,
```
Comment (adriandumavitagroup) [inline src/app/layout/components/app-layout/app-layout.component.spec.ts:67]: delete this

File: src/app/layout/components/app-layout/app-layout.component.ts [hunk 2]
```diff
@@ -37,10 +41,17 @@ export class AppLayoutComponent implements OnInit, OnDestroy {
     this.matcher.addEventListener('change', (event) => {
       this.isSmallDeviceListener(event)
     })
+
+    this.subscriptions.add(
+      this.profileService.getUnapprovedUser().subscribe((response: any) => {
+        this.unapprovedUser = response === true
```
Comment (adriandumavitagroup) [inline src/app/layout/components/app-layout/app-layout.component.ts:47]: why not just this.unapprovedUser = response  ?

File: src/app/modules/profile/components/profile/profile.component.ts [hunk 3]
```diff
@@ -50,6 +50,7 @@ export class ProfileComponent implements OnInit {
   })
 
   ngOnInit(): void {
+    console.log('rrr')
```
Comment (adriandumavitagroup) [inline src/app/modules/profile/components/profile/profile.component.ts:53]: delete this

File: src/app/layout/components/app-layout/app-layout.component.spec.ts (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/app/layout/components/app-layout/app-layout.component.spec.ts]: Done.

File: src/app/layout/components/app-layout/app-layout.component.ts (hunk 2 above)
Comment (sandorbanvitagroup) [reply src/app/layout/components/app-layout/app-layout.component.ts]: Yep, no reason for that to test

File: src/app/modules/profile/components/profile/profile.component.ts (hunk 3 above)
Comment (sandorbanvitagroup) [reply src/app/modules/profile/components/profile/profile.component.ts]: Done.

File: src/app/core/services/profile/profile.service.ts [hunk 4]
```diff
@@ -27,8 +27,9 @@ import { IUserProfile } from '../../../shared/models/user/user-profile.interface
 export class ProfileService {
   private userProfile = {} as IUserProfile
   private userProfileSubject$ = new BehaviorSubject(this.userProfile)
+  private subject = new Subject<any>()
```
[the remark this answers is not in the document]
Comment (sandorbanvitagroup) [reply src/app/core/services/profile/profile.service.ts]: Done.

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**